app/src/panels/time-series/time-series.vue

```diff
@@ -107,6 +107,21 @@ export default defineComponent({
 		const chartEl = ref();
 		const chart = ref<ApexCharts>();
 
+		const yAxisRange = computed(() => {
+			let min = isNil(props.min) ? undefined : Number(props.min);
+			let max = isNil(props.max) ? undefined : Number(props.max);
+
+			if (max && !min) {
```

Comment:
Actually, we should do a more explicit "is undefined" check here. the number `0` could be a valid option for the value, but is also considered falsy in JS, so would trigger these if statements wrong